3p-ecosystem/src/main/java/com/google/homesampleapp/screens/thread/OtbrHttpClient.kt

```diff
@@ -103,7 +101,7 @@ object OtbrHttpClient {
           urlConnection.doOutput = true
           try {
             val outputStream = urlConnection.outputStream
```

Comment:
kotlin has a [nice extension function for closeable resources](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/use.html) to automatically close them after use, this could be:

```
urlConnection.outputStream.use { outputStream ->
    outputStream.write(postPayload.toByteArray())
    outputStream.flush()
}
```

This way you can drop the `try / catch` blocks.

Reply:
Thanks for the tip. Fixed.